modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -12,13 +12,137 @@ endif::[]
+
+You can define scopes in three different places:
+
+* In the universal set definition, `defaultScopes`, where you define all possible scopes.
+Example:
++
+[source,xml,linenums]
+----
+<oauth2-provider:config
+  scopes="CONTRACTOR, PUBLIC_READ, EMPLOYEES_ONLY, WRITE"
+  >
+  ...
+</oauth2-provider:config>
+----
+* In the `/validate` validation endpoint, to enforce a particular subset of the previously defined scopes.
+* In the Mule OAuth 2.0 Access Token Enforcement policy, where you can enforce scopes by specifying a space-separated list.
++
+Specifying more than one scope enforces the `OR` logic. If the token to be validated has associated at least one of the requested scopes, the policy validates the token successfully.
+
+Example of using scopes in Mule 4.2.x or later:
+
+[source,xml,linenums]
+----
+...
+    <oauth2-provider:config name="external-oauth2-provider"
+    							resourceOwnerSecurityProvider="resourceOwnerSecurityProvider"
+    							clientSecurityProvider="clientSecurityProvider"
+    							supportedGrantTypes="AUTHORIZATION_CODE,IMPLICIT,RESOURCE_OWNER_PASSWORD_CREDENTIALS,CLIENT_CREDENTIALS"
+    							listenerConfig="https.listener"
+    							scopes="CONTRACTOR, PUBLIC_READ, EMPLOYEES_ONLY, WRITE"
+                                defaultScopes="CONTRACTOR, PUBLIC_READ, EMPLOYEES_ONLY, WRITE"
+                                clientStore="clientObjectStore">
+    		<oauth2-provider:token-config path="/access-token" tokenTtl="5" tokenTtlTimeUnit="SECONDS" tokenStore="tokenObjectStore">
+                <oauth2-provider:refresh-token-strategy>
+                    <oauth2-provider:single-refresh-token objectStore="refreshTokenObjectStore"/>
+                </oauth2-provider:refresh-token-strategy>
+            </oauth2-provider:token-config>
+    		<oauth2-provider:authorization-config path="/authorize" authorizationCodeStore="authorizationCodeObjectStore"/>
+    	</oauth2-provider:config>
+
+        <flow name="validateToken">
+            <http:listener path="/validate" config-ref="https.listener"/>
+            <oauth2-provider:validate-token scopes="#[['PUBLIC_READ']]" config-ref="external-oauth2-provider" accessToken="#[attributes.queryParams.access_token]"/>
+            <error-handler>
+                <on-error-continue type="OAUTH2-PROVIDER:TOKEN_UNAUTHORIZED">
+                    <set-payload value="UNAUTHORIZED_TOKEN"/>
+                </on-error-continue>
+            </error-handler>
+        </flow>
+...
+----
+
+For complete information, see https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform].
+
+== Client Application Cache
+
+To prevent any possible downtime of the Mule OAuth 2.0 provider due to errors when connecting to Anypoint Platform, the Mule OAuth Client Store caches each valid Client Application for which a token is requested.
```

Comment:
I see no reason to capitalize client store or client application, which are generic things:
```suggestion
To prevent any possible downtime of the Mule OAuth 2.0 provider due to errors when connecting to Anypoint Platform, the Mule OAuth client store caches each valid client application for which a token is requested.
```

Reply:
Done! Thanks!